src/schedule/Schedule.ts

```diff
@@ -47,18 +50,27 @@ export class Schedule extends LogEmitter {
    * the scheduler has to be started again to pick up the change.
    *
    * @param momoJob the job to define
+   * @returns true if jobs was defined, false if the job was invalid
    */
-  public async define(momoJob: MomoJob): Promise<void> {
-    const job = withDefaults(momoJob);
+  public async define(momoJob: MomoJob): Promise<boolean> {
```

Comment:
can we remove the 'public' everywhere?

Reply:
All those functions are part of the public interface. Why would you like to remove public?

Reply:
yes you're right, I don't remember what I thought :D